contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -15,6 +15,10 @@ contract RaidenMicroTransferChannels {
     // Contract semantic version
     string public constant version = '1.0.0';
 
+    // We temporarily limit total token deposits in a channel to 100 tokens with 18 decimals.
+    // This is just for the bug bounty release.
+    uint256 public channel_deposit_bugbounty_limit = 10 ** 18 * 100;
```

Comment:
Please make it a constant since it's not supposed to ever be changed for the lifetime of this contract